kustomize/README.md

```diff
@@ -36,10 +38,10 @@ If you are enabling the **Memorystore** deployment variation, first complete the
     cd kustomize
     ```
 
-1. Edit the base level `kustomization.yaml` so that it is uses the intended components.
+1. Edit the base level `kustomization.yaml` so that it uses the components you wish to use.
 
     ```
-    vim kustomization.yaml
+    nano kustomization.yaml
```

Comment:
My rationale for this change: I've used `nano` instead of `vim` because `vim` is (notoriously) user-hostile (for new users).